client/extensions/woocommerce/app/order/order-details.js

```diff
@@ -24,13 +27,57 @@ class OrderDetails extends Component {
 	}
 
 	state = {
-		showRefundDialog: false,
+		status: this.props.order.status || false,
 	}
 
-	toggleRefundDialog = () => {
-		this.setState( {
-			showRefundDialog: ! this.state.showRefundDialog,
-		} );
+	updateStatus = ( event ) => {
+		this.setState( { status: event.target.value } );
+		// Send the order back to the parent component
+		this.props.onUpdate( { status: event.target.value } );
+	}
+
+	renderStatus = () => {
+		const { order, translate } = this.props;
+		const classes = `order__status is-${ order.status }`;
+		const statuses = [ {
+			value: 'pending',
+			name: translate( 'Pending payment' ),
+		}, {
+			value: 'processing',
+			name: translate( 'Processing' ),
+		}, {
+			value: 'on-hold',
+			name: translate( 'On Hold' ),
+		}, {
+			value: 'completed',
+			name: translate( 'Completed' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **52** times:
`translate( '&laquo; completed' )` *ES Score: 10*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/woo-orders-save-status)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).